docs/Config-Specification.rst

```diff
@@ -517,17 +517,17 @@ interfaces:
-|                           |     iface: myvlan.20                              |                                                                                            |          |
-|                           |     method: manual                                |                                                                                            |          |
-|                           |     vlan_raw_device: eth0                         |                                                                                            |          |
-|                           |                                                   |                                                                                            |          |
-|                           |   - label: bridge1                                |                                                                                            |          |
-|                           |     description: bridge interface 1               |                                                                                            |          |
-|                           |     iface: br1                                    |                                                                                            |          |
-|                           |     method: static                                |                                                                                            |          |
+|                           |                                                   | .. _interfaces_ubuntu_rename_notes:                                                        |          |
+|                           |   - label: vlan2                                  |                                                                                            |          |
+|                           |     description: vlan interface 2                 | | Notes:                                                                                   |          |
+|                           |     iface: myvlan.20                              | |   If 'rename: true' in                                                                   |          |
+|                           |     method: manual                                |     `node_templates: physical_interfaces: pxe/data <physical_ints_os_>`_ then the *iface*  |          |
+|                           |     vlan_raw_device: eth0                         |     value will be used to rename the interface.                                            |          |
+|                           |                                                   | |                                                                                          |          |
+|                           |   - label: bridge1                                | |   If 'rename: false' in                                                                  |          |
```

Comment:
Does the statement "If the iface value is referenced in any other interface definition it will also be replaced" only apply when rename is 'false'?

Reply:
Yes. Nothing is changed if rename=true.